website/blog/2019-05-01-react-native-at-f8-and-podcast.md

```diff
@@ -0,0 +1,38 @@
+---
+title: React Native at F8 and Open Source Podcast
+author: Christoph Nakazawa
+authorTitle: Engineer at Facebook
+authorURL: https://twitter.com/cpojer
+authorImageURL: https://avatars2.githubusercontent.com/u/13352?s=460&v=4
+authorTwitter: cpojer
+category: announcements
+---
+
+This week, [Eli White](https://twitter.com/Eli_White) gave a talk at [F8 2019](https://developers.facebook.com/videos/2019/mobile-innovation-with-react-native-componentkit-and-litho/) about React Native in Facebook's Android and iOS applications. We are very excited to tell you what we've been up to for the past two years and what we are doing next. Check out [the video on the Facebook's developer website](https://developers.facebook.com/videos/2019/mobile-innovation-with-react-native-componentkit-and-litho/):
```

Comment:
```suggestion
This week, [Eli White](https://twitter.com/Eli_White) gave a talk at [F8 2019](https://developers.facebook.com/videos/2019/mobile-innovation-with-react-native-componentkit-and-litho/) about React Native in Facebook's Android and iOS applications. We are excited to share what we've been up to for the past two years and what we're doing next.

Check out the video on [Facebook's developer website](https://developers.facebook.com/videos/2019/mobile-innovation-with-react-native-componentkit-and-litho/):
```

- Remove "very"
- Switch to "show" instead of "tell"
- Add whitespace
- Format link a bit differently

Reply:
Should the link be on “about React Native...” instead of “at F8”?